packages/rest-client/src/generators/bump-version/schema.json

```diff
@@ -6,21 +6,26 @@
     "description": "The @ensono-stacks/rest-client:bump-version generator will bump the version of any generated endpoints",
     "type": "object",
     "properties": {
-        "endpointPath": {
+        "name": {
             "type": "string",
-            "description": "The path to your endpoint.",
-            "x-prompt": "What is the path of your endpoint?"
+            "description": "The endpoint name you want to bump to the specified version.",
+            "x-prompt": "What is the endpoint you want to bump to the specified version?",
+            "$default": {
+                "$source": "argv",
+                "index": 0
+            }
         },
-        "endpoint": {
+        "directory": {
             "type": "string",
-            "description": "The endpoint you want to bump to the specified version.",
-            "x-prompt": "What is the endpoint you want to bump to the specified version?"
+            "description": "The directory where the endpoint lib is placed.",
+            "alias": "d"
```

Comment:
Does `d` alias work for you? Seems like they decided against changing this within Nx and kept it to alias dry-run. Recommended is `dir`
https://github.com/nrwl/nx/issues/6571
https://github.com/nrwl/nx/pull/6596

Let me know what you think :)

Reply:
Indeed! Pushed fix.